source/broadband/cosa_webpa_dml.c

```diff
@@ -175,6 +177,69 @@ Webpa_SetParamStringValue
 		   }
         	}
 
+	if( AnscEqualString(ParamName, "ConnectedClientNotify", TRUE))
+        {
+		if( GET_CURRENT_WRITE_ENTITY() == CCSP_COMPONENT_ID_WEBCONFIG )
+		{
+	  	char notif[20] = "";
+		WDMP_STATUS ret = WDMP_FAILURE;
+		param_t *attArr = NULL;
+
+		if(pString != NULL)
+		{
+			if(strcmp(pString, "enabled")==0)
+			{
+				snprintf(notif, sizeof(notif), "%d", 0);
+			}
+			else if(strcmp(pString, "disabled")==0)
+			{
+				snprintf(notif, sizeof(notif), "%d", 1);
+			}
+			else
+			{
+				WalError("Invalid apply status in ACK event\n");
+				return FALSE;
+			}
+					
+			if(CONN_CLIENT_PARAM == NULL)
+			{
+				WalError("Failed to get connected client paramName\n");
+				return FALSE;
+			}
+			attArr = (param_t *) malloc(sizeof(param_t));
+			if(attArr !=NULL)
+			{
+				memset(attArr,0,sizeof(param_t));
+				attArr[0].value = (char *) malloc(sizeof(char) * 20);
+				snprintf(attArr[0].value, 20, "%s", notif);
+				attArr[0].name = CONN_CLIENT_PARAM;
+				attArr[0].type = WDMP_INT;
+				setAttributes(attArr, 1, NULL, &ret);
+				if (ret != WDMP_SUCCESS)
+				{
+					WalError("setAttributes failed for parameter : %s notif:%s ret: %d\n", CONN_CLIENT_PARAM, notif, ret);
```

Comment:
@sadhyama in case of setAttributes error should we mark this parameter set also as failure?